crypto/nrf_oberon/include/ocrypto_ecdh_p256.h

```diff
@@ -14,19 +21,25 @@ extern "C" {
 #include <stdint.h>
 
 
-// ecdh P-256 public key r = n * p
-// r[64]: the resulting public key
-// s[32]: the secret key
-// r may be same as s
-// returns 0 if s is a legal secret key
+/** ecdh P-256 public key r = n * p
+ *
+ * @param[out]  r   The resulting public key
+ * @param       s   The secret key
+ *
+ * @remark @p r may be the same as @p s
+ * @returns 0 if s is a legal secret key
+ */
 int ocrypto_ecdh_p256_public_key(uint8_t r[64], const uint8_t s[32]);
 
-// ecdh P-256 common secret
-// r[32]: the resulting secret
-// s[32]: the secret key
-// p[64]: the public key
-// r may be same as s or p
-// returns 0 if s is a legal secret key and p is a legal public key
+/** ecdh P-256 common secret
```

Comment:
Is this the same as shared secret? If so say "ecdh P-256 common (shared) secret"